README.md

```diff
@@ -135,7 +135,9 @@ An example of run the acceptance tests follow the instructions [here].
 
 ## Support
 
-Please log tickets and issues in our [JIRA tracker][JIRA]. A [mailing list](https://groups.google.com/forum/?fromgroups#!forum/puppet-users) is available for asking questions and getting help from others.
+Please log issues in [GitHub issues][issues]. Check out [CONTRIBUTING.md][contributing] for tips on writing _the best_ issues.
+
+A [mailing list](https://groups.google.com/forum/?fromgroups#!forum/puppet-users) is available for asking questions and getting help from others.
 
 There is also an active #puppet channel on the Freenode IRC network.
```

Comment:
We should mention the [community Slack](https://slack.puppet.com) as well. It's at least as active as IRC.

Reply:
good call. I thought about this, which channel should we direct them to?

Reply:
i will add that. also, jean and I discussed removing the IRC just because we weren't aware of anyone on the team watching freenode, what do you think about that @scotje ?